test/functional/zmq_test.py

```diff
@@ -7,12 +7,18 @@
 import os
 import struct
 
+from codecs import encode
+
+from test_framework.mininode import dashhash
 from test_framework.test_framework import BitcoinTestFramework, SkipTest
 from test_framework.util import (assert_equal,
                                  bytes_to_hex_str,
                                  hash256,
                                 )
 
+def dashhash_helper(b):
```

Comment:
I would probably have put a descriptive comment for this func here. Not immediately clear unless you understand what is going on. `# Accepts raw 80-byte block header, runs Dash PoW hash and reverses the bytes per convention` or something like that. But not a huge deal.